servers/audio_server.cpp

```diff
@@ -1361,7 +1375,7 @@ void AudioServer::init_channels_and_buffers() {
 void AudioServer::init() {
 	channel_disable_threshold_db = GLOBAL_DEF_RST("audio/buses/channel_disable_threshold_db", -60.0);
 	channel_disable_frames = float(GLOBAL_DEF_RST(PropertyInfo(Variant::FLOAT, "audio/buses/channel_disable_time", PROPERTY_HINT_RANGE, "0,5,0.01,or_greater"), 2.0)) * get_mix_rate();
-	buffer_size = 512; //hardcoded for now
+	buffer_size = 512; // Hardcoded for now. This would be really nice to have as a project setting because currently it limits audio latency to an absolute minimum of 11ms with default mix rate, but there's some additional work required to make that happen. See TODOs in `_mix_step_for_channel`.
```

Comment:
In reality most professional audio software I know uses 256 (~6ms at 44100) as the absolute minimum block size because this is considered the limit that you can tell apart if say, you are creating a synthesizer and will be playing notes received via Midi. Vastly most people can't really tell the difference between 12 and 6ms.

For game content, 12ms latency should be way more than enough.

That said, I think our problem here is different and its that we should make it customizable not as a block size but as a minimum latency size in ms. This way we can have a default of say, 10ms, and then the audio server converts this to audio frames based on the sampling rate and it finds the nearest power of 2 immediately above.

This should be ideal because it is very common that some devices will open audio in stuff like 192khz, for which 512 audio frames block is an absolute waste.